src/tools/rbd/action/ObjectMapCheck.cc

```diff
@@ -0,0 +1,74 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
I'd still like to see all the object map commands handles within "src/tools/rbd/action/ObjectMap.cc"